cli/src/wallet.rs

```diff
@@ -65,7 +65,8 @@ pub enum WalletCommand {
     Cancel(Pubkey),
     Confirm(Signature),
     AuthorizeVoter(Pubkey, Keypair, Pubkey),
-    CreateVoteAccount(Pubkey, Pubkey, u8, u64),
+    AuthorizeWithdrawer(Pubkey, Keypair, Pubkey),
```

Comment:
Can you please use the struct style here, like `ShowAccount`.   `CreateVoteAccount` for example is completely unreadable as a tuple:
```rust
CreateVoteAccount(🍝, 🍝, 🍝, 🍝, u8, u64),
```

Reply:
you and @CriesofCarrots have the same good ideas
:thumbsup:

I'm thinking that after today (#6074) that maybe we outlaw tuples that have any more than 1 of the same type...

Reply:
Yeah.  I'd lobby to go even stronger and outlaw tuples in any public API.  They're just error prone and always the start of 🍝

Reply:
man it took me forever to get that emoji resolved